tools/runners/test262-harness.py

```diff
@@ -452,12 +461,12 @@ def report_outcome(self, long_format):
     def write_output(self, target):
         out = self.stdout.strip()
         if out:
-            target.write("--- output --- \n %s" % out)
+            write_text_to(target, u"--- output --- \n%s\n" % out)
```

Comment:
If suggestion given at https://github.com/jerryscript-project/jerryscript/pull/4390#pullrequestreview-570801294 is applied (i.e., `target` is opened with a proper encoding), then this change (i.e., wrapping the write into a function) may not even be needed. Please, check and confirm.

Reply:
> If suggestion given at [#4390 (review)](https://github.com/jerryscript-project/jerryscript/pull/4390#pullrequestreview-570801294) is applied (i.e., `target` is opened with a proper encoding), then this change (i.e., wrapping the write into a function) may not even be needed. Please, check and confirm.

Does codecs possible to open stdout/stderr?
Because this target is file or stdout

Reply:
Stdout or stderr cannot/shouldn't be re-opened, but codecs could still help. Stdout and stderr can be wrapped at the start of the program with something like:
```
(out_stream, err_stream) = (sys.stdout.buffer, sys.stderr.buffer) if sys.version_info.major >= 3 else (sys.stdout, sys.stderr)

if sys.stdout.encoding != 'UTF-8':
    sys.stdout = codecs.getwriter('utf-8')(out_stream, 'ignore')
if sys.stderr.encoding != 'UTF-8':
    sys.stderr = codecs.getwriter('utf-8')(err_stream, 'ignore')
```
(disclaimer: untested)